cap_anndata/cap_anndata.py

```diff
@@ -269,6 +332,22 @@ def read_uns(self, keys: List[str] = None) -> None:
                 source = self._file[f"uns/{key}"]
                 self.uns[key] = read_elem(source)
 
+    def _link_layers(self) -> None:
+        if self._layers is None:
+            self._layers = CapAnnDataDict()
+        if "layers" in self._file.keys():
+            group = self._file["layers"]
+            for entity_name in group.keys():
+                entity = group[entity_name]
+                if isinstance(entity, h5py.Dataset):
+                    # dense array
+                    self._layers[entity_name] = entity
+                else:
+                    # sparse array
+                    self._layers[entity_name] = ad.experimental.sparse_dataset(entity)
```

Comment:
Please use just `sparse_dataset`, it was explicitly imported

Reply:
done